README.md

```diff
@@ -1,129 +1,108 @@
 [![NPM downloads](https://img.shields.io/npm/dt/@brightspace-ui/core.svg)](https://www.npmjs.com/package/@brightspace-ui/core)
 [![Greenkeeper badge](https://badges.greenkeeper.io/BrightspaceUI/core.svg)](https://greenkeeper.io/)
 [![Build status](https://travis-ci.com/BrightspaceUI/core.svg?branch=master)](https://travis-ci.com/BrightspaceUI/core)
 
-A collection of accessible, free, open-source web components for building Brightspace applications.
+A collection of accessible, free, open-source web components and tools for building Brightspace applications.
 
 ## Installation
 
-To install from NPM:
-
 ```shell
 npm install @brightspace-ui/core
 ```
 
-## Components
-
-* [Buttons](components/button/): normal, primary, icon and subtle buttons
-* [Colors](components/colors/): color palette
-* [Dialogs](components/dialog/): generic and confirmation dialogs
-* [Icons](components/icons/): iconography SVGs and web components
-* [Inputs](components/inputs/): text, search, select, checkbox and radio inputs
-* [Links](components/link/): link component and styles
-* [List](components/list/): list and list-item components
-* [Loading Spinner](components/loading-spinner/): loading-spinner components
-* [Meter](components/meter/): linear, radial, circle meter web components
-* [More/less](components/more-less/): constrain long bits of content
-* [Off-screen](components/offscreen/): component and styles for positioning content off-screen
-* [Status Indicator](components/status-indicator/): status-indicator components
-* [Typography](components/typography/): typography styles and components
-
-## Helpers
-
-* [Helpers](helpers/): helpers for composed DOM, unique ids, etc.
-
-## Mixins
-
-* [ArrowKeysMixin](mixins/arrow-keys-mixin.md): manage focus with arrow keys
-* [AsyncContainerMixin](mixins/async-container/): manage collective async state
-* [LocalizeMixin](mixins/localize-mixin.md): localize text in your components
-* [ProviderMixin](mixins/provider-mixin.md): provide and consume data across elements in a DI-like fashion
-* [RtlMixin](mixins/rtl-mixin.md): enable components to define RTL styles
-* [VisibleOnAncestorMixin](mixins/visible-on-ancestor-mixin.md): display element on-hover of an ancestor
-
-## Templates
-
-* [PrimarySecondaryTemplate](templates/primary-secondary): Two Panel (primary and secondary) page template with header and optional footer
-
-## Usage
-
-### Structure
-
-`/components/` - Web components for use in your composite components or application (ex. buttons, links, etc)
-
-`/helpers/` - Helpers for use in your web components or mixins (ex. composed DOM helpers)
-
-`/mixins/` - Mixins for use in your web components (ex. localize, RTL, etc)
-
-`/templates/` - Page templates to provide common ways to arrange content on a page
-
-`/test/` - Shared styles for unit tests
-
-`/tools/` - Development tools for building, testing, etc.
-
-## Developing, Testing and Contributing
+## README Index
+
+* Components
```

Comment:
Just want to confirm the style you are going for here with the `*` and `-` -  it's appearing as a single bullet point list with a bit of space between the bullet for components and then the components (same with helpers and mixins)

Reply:
Hmm, nope that's not right. Last time I trust an online markdown resource -- will fix, thanks!

Reply:
k fixed